Framework/Algorithms/src/FindPeaks.cpp

```diff
@@ -1087,7 +1087,7 @@ int FindPeaks::findPeakBackground(const MatrixWorkspace_sptr &input,
         "No 7th column for use FindPeakBackground result or not. ");
 
   if (peaklisttablews->rowCount() > 0) {
-    int fitresult = peaklisttablews->Int(0, 6);
+    fitresult = peaklisttablews->Int(0, 6);
```

Comment:
@FedeMPouzols is right. Unshadowing that variable is bad. I haven't had the time to track down why it is so important, but the fits are significantly better with it there.